pkg/controller/volume/pvprotection/pv_protection_controller_test.go

```diff
@@ -210,7 +210,7 @@ func TestPVProtectionController(t *testing.T) {
 
 		// Create the controller
 		logger, _ := ktesting.NewTestContext(t)
-		ctrl := NewPVProtectionController(logger, pvInformer, client)
+		ctrl := NewPVProtectionController(logger, pvInformer, client, "persistentvolume-protection-controller")
```

Comment:
shouldn't we pass in the constants to all these test invocations as well? hard-coding to a string gives the appearance that the name doesn't matter, when we actually want them to be consistent, right?

Reply:
@liggitt those constants are defined in `cmd/kube-controller-manager/names`, do we want to import this package into `pkg` and `tests` dir?

Reply:
It is better not to depend on `/cmd`: https://github.com/kubernetes/kubernetes/blob/2fda3e4a650d8e729c509609aa865f0b6d843b4e/pkg/.import-restrictions#L8 

It would be nicer to use a constant, but I think it doesn't pose a big issue to use the hard-coded strings in the tests.

Hypothetically, does it make sense to modify the name to distinguish the test names? E.g. "persistentvolume-protection-controller-test"?

Reply:
> Hypothetically, does it make sense to modify the name to distinguish the test names? E.g. "persistentvolume-protection-controller-test"?

It's better to have the same value. It won't make developers confused when they look at the log file.

Reply:
ok